.github/ISSUE_TEMPLATE/FEATURE_REQUEST.md

```diff
@@ -10,7 +10,9 @@ As a [type of user], I want [an action] so that [a benefit/a value].
 
 **Acceptance Criteria**
 <!-- What should be done for the issue to be considered completed. Add checklist item(s) and remove examples not applicable -->
-- [ ] The documentation is updated accordingly
+- [ ] <!-- Issue specific criteria -->
```

Comment:
I also get a bit confused here, so maybe it's good to rephrase a bit. We could add something similar to the comment for bug reports?
> " These items should be updated to match this specific PR. Add criteria item(s) and remove examples not applicable "

Reply:
Not really sure what you are referring to in the bug report, and if you would like to keep the new empty checklist item or not. Could you make a commit with the changes you propose? And I think maybe `PR` should be replaced with `issue` right?

Reply:
```suggestion
<!-- These items should be updated to match this specific issue. Add criteria item(s) and remove examples not applicable -->
- [ ] <!-- Issue specific criteria -->
```

I copied the second sentence in the comment from your new version of the `.github/ISSUE_TEMPLATE/BUG_REPORT.md`, that what was I meant by "the bug report" :). 

For the new empty checklist item, I think it's a nice way to encourage people to write their own items, so I'm up for keeping it.

Reply:
In the current commit c1fedec4e7a1653d464e5e82d1f86681f73f34e7 we already have

```
**Acceptance Criteria**
<!-- What should be done for the issue to be considered completed. Add checklist item(s) and remove examples not applicable -->
```
So maybe rewrite that comment instead of adding another one below that? :)

Reply:
Ok, sorry, I missed that.
I think your current version is really good, I would only want to rephrase the check list item slightly. How about
```
- [ ] Placeholder for issue specific criterion
```
If it's not put in an html comment, it looks lees like an heading for the rest of the items. But I'm fine having it as a comment to. I'm also fine with having plural form of criterion, I can't decide which is less confusing here.

Reply:
Good suggestion! Added it in 8265cf5bd0eb113b3cfdbb76f3176d3bbdfd9ce3